src/tactic/norm_cast.lean

```diff
@@ -437,7 +437,7 @@ add_tactic_doc
   decl_names := [`tactic.interactive.norm_cast, `tactic.interactive.rw_mod_cast,
                  `tactic.interactive.apply_mod_cast, `tactic.interactive.assumption_mod_cast,
                  `tactic.interactive.exact_mod_cast, `tactic.interactive.push_cast],
-  tags       := ["coercions", "simplification"] }
+  tags       := ["coercions", "simplification", "normalization"] }
```

Comment:
Do `exact_mod_cast` and `assumption_mod_cast` fall under "finishing" and/or "lemma application"?